src/generators/static/packageFileGenerator.ts

```diff
@@ -88,7 +88,7 @@ function restLevelPackage(packageDetails: PackageDetails) {
     },
     dependencies: {
       "@azure-rest/core-client": "1.0.0-beta.4",
-      "@azure/core-auth": "^1.1.4",
+      "@azure/core-auth": "^1.3.0",
       "@azure/core-rest-pipeline": "^1.0.4"
```

Comment:
`@azure/core-rest-pipeline` should be updated to "^1.1.0" as well - can we do it as part of this PR or a separate PR?

Reply:
I can combine it with this PR. Will update it shortly

Reply:
1.1.0 of core-rest-pipeline is not published yet...

Reply:
Spoke offline with Maor. Will keep core-rest-pipeline to 1.0.4 all through for this week.